frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -16,11 +16,18 @@ import TooltipWithCollection from '../../components/TooltipWithCollection';
 import Tooltip from '../../components/Tooltip';
 import TableHeader from '../../components/TableHeader';
 import { cleanupLocalStorage } from '../ActivityReport';
+import UserContext from '../../UserContext';
+
+const userIsAnApprover = (id, approvers) => {
```

Comment:
what if I'm the creator and an approver?

Reply:
ah yup, 27168c86c9375601956a0a564ddc4d275105ece0